CollAction/Controllers/ProjectsController.cs

```diff
@@ -249,6 +264,8 @@ public ViewResult StartInfo()
                 project.BannerImage = await manager.UploadFormFile(editProjectViewModel.BannerImageUpload, Guid.NewGuid().ToString() /* unique filename */);
             }
 
+            await project.SetTags(_context, editProjectViewModel.Hashtag?.Split(';') ?? new string[0]);
```

Comment:
Doesn't matter now, but we should probably think up a better way to pass lists from the frontend to the backend (json, something buildin in asp.net core).